custom_model_runner/datarobot_drum/drum/push.py

```diff
@@ -149,7 +149,7 @@ def _setup_training_validation(config, options):
     options.negative_class_label = None
     options.positive_class_label = None
     options.target_csv = None
-    options.target = config["validation"]["targetName"]
+    options.target = config["validation"].get("targetName")
```

Comment:
what happens to the `raw_args_for_docker` command when target here is None/empty?

Reply:
...I guess it must work, since there seems to be an anomaly test case below